### PR TITLE
refactor input view for ccc

### PR DESCRIPTION
**Pull request by pbochis**
- make all the tests run together(if not empty)

### CODE REVIEW

**Comment by pbochis** [comment]
depends on [this](https://github.com/coduno/platform/pull/80)

**Comment by flowlo** [comment]
@pbochis, if you do not reference using a link, but directly, this will be rendered nicely: coduno/platform#80

File: app/elements/coder/input-view/input-view.html [hunk 1]
```diff
@@ -1,4 +1,9 @@
 <link rel="import" href="input-view-element.html">
+<link rel="import" href="/bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="/bower_components/paper-icon-button/paper-icon-button.html">
```
Comment (victorbalan) [inline app/elements/coder/input-view/input-view.html:3]: double import

**Comment by victorbalan** [comment]
#183

**Pull request closed without merging**